youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+
+#: youtube_dl_gui/mainframe.py:115
+msgid "View Log"
+msgstr "צפה ביומן רישום"
+
+#: youtube_dl_gui/mainframe.py:117
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "הוריד בהצלחה את {0} כתובות האתרים תוך {1} ימים {2} שעות {3} דקות {4} שניות"
+
+#: youtube_dl_gui/mainframe.py:119
+msgid "Downloads completed"
+msgstr "ההורדה הושלמה"
+
+#: youtube_dl_gui/mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "סה"כ התקדמות: {0:.1f}% | בהמתנה ({1}) בהפסקה ({2}) פעילים ({3}) הושלמו ({4}) בשגיאה ({5})"
+
+#: youtube_dl_gui/mainframe.py:121
+msgid "Stopping downloads"
+msgstr "עוצר הורדות"
+
+#: youtube_dl_gui/mainframe.py:122
+msgid "Downloads stopped"
+msgstr "ההורדה נעצרה"
+
+#: youtube_dl_gui/mainframe.py:123
+msgid "You need to provide at least one URL"
+msgstr "עליך לספק לפחות כתובת אתר אחת"
+
+#: youtube_dl_gui/mainframe.py:124
+msgid "Downloads started"
+msgstr "ההורדה התחילה"
+
+#: youtube_dl_gui/mainframe.py:125
+msgid "Choose Directory"
+msgstr "בחר תיקייה"
+
+#: youtube_dl_gui/mainframe.py:127
+msgid "Download in progress. Please wait for all downloads to complete"
+msgstr "ההורדה מתבצעת. המתן עד שכל ההורדות יושלמו"
+
+#: youtube_dl_gui/mainframe.py:128
+msgid "Update already in progress"
+msgstr "העדכון כבר מתבצע"
+
+#: youtube_dl_gui/mainframe.py:130
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "מוריד את הגירסא האחרונה של youtube-dl. אנא המתן"
+
+#: youtube_dl_gui/mainframe.py:131
+msgid "Youtube-dl download failed [{0}]"
+msgstr "הורדת youtube-dl נכשלה [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "youtube-dl הורד בהצלחה"
+
+#: youtube_dl_gui/mainframe.py:134
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr "פתיחת התיקייה נכשלה: '{dir}'. הנתיב שצוין אינו קיים"
+
+#: youtube_dl_gui/mainframe.py:136
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr "שגיאה בעת כיבוי. ודא שהקלדת את הסיסמה הנכונה"
+
+#: youtube_dl_gui/mainframe.py:138
+msgid "Shutting down system"
+msgstr "כיבוי המערכת"
+
+#: youtube_dl_gui/mainframe.py:141
+msgid "Extension"
+msgstr "סיומת"
+
+#: youtube_dl_gui/mainframe.py:142
+msgid "Size"
+msgstr "גודל"
+
+#: youtube_dl_gui/mainframe.py:143
+msgid "Percent"
+msgstr "אחוז"
+
+#: youtube_dl_gui/mainframe.py:144
+msgid "ETA"
+msgstr ""
+
+#: youtube_dl_gui/mainframe.py:145
+msgid "Speed"
+msgstr "מהירות"
+
+#: youtube_dl_gui/mainframe.py:146
+msgid "Status"
+msgstr "מצב"
+
+#: youtube_dl_gui/mainframe.py:235
+msgid "Get URL"
+msgstr "קבל כתובת אתר"
+
+#: youtube_dl_gui/mainframe.py:236
+msgid "Get command"
+msgstr "קבל פקודה"
+
+#: youtube_dl_gui/mainframe.py:237
+msgid "Open destination"
+msgstr "פתח יעד"
+
+#: youtube_dl_gui/mainframe.py:238
+msgid "Re-enter"
+msgstr "הזן מחדש"
+
+#: youtube_dl_gui/mainframe.py:458
+msgid "Resume"
+msgstr "התחל מחדש"
```

Comment:
Google translates this to "Start over" which is like the opposite of "Resume" (continue from were you stopped).
Are you sure this is correct?

Reply:
I change this in the fix commit